src/pages/TransactionDuplicate/Confirmation.tsx

```diff
@@ -109,7 +122,13 @@ function Confirmation() {
                             <Button
                                 text={translate('common.confirm')}
                                 success
-                                onPress={mergeDuplicates}
+                                onPress={() => {
+                                    if ((isAdmin || isApprover) && !isActionOwner) {
```

Comment:
I would change this condition to be `!isActionOwner` since everyone else should use ResolveDuplicate